src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,50 @@
-export const Tabs = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { Tab } from '../../types/Tab';
+
+interface Props {
+  tabs: Tab[];
+  currentTab: Tab;
+  onSelect: (tab: Tab) => void;
+}
+export const Tabs: React.FC<Props> = (
+  {
+    tabs,
+    currentTab,
+    onSelect,
+  },
+) => {
```

Comment:
I think you can't pass the test, because you don't have checker if your page changing
```suggestion
) => {
  const selectedTab = tabs.find(tab => tab.id === currentTab.id) || tabs[0];

   const handleClick = (tab: Tab) => {
     if (tab.id !== currentTab.id) {
       onSelect(tab);
     }
   };
```